src/registrar/config/settings.py

```diff
@@ -297,12 +297,16 @@
 
 # Content-Security-Policy configuration
 # this can be restrictive because we have few external scripts
-allowed_sources = ("'self'",)
+allowed_sources = "'self'"
```

Comment:
**suggestion (blocking)**: It is surprising, but allowed_sources actually needs to be a one-element tuple here instead of a single string. The parentheses and the comma are required.
```suggestion
allowed_sources = ("'self'",)
```

Reply:
Oh wow...I actually don't remember removing the tuple enclosure...I wonder if linting did.....Thanks for the catch